src/pykeen/hpo/hpo.py

```diff
@@ -802,7 +802,6 @@ def _set_study_dataset(
     study: Study,
     *,
     dataset: Union[None, str, Dataset, Type[Dataset]] = None,
-    dataset_kwargs: Optional[Mapping[str, Any]] = None,
```

Comment:
can you explain what's changed here and why?

Reply:
I removed it because it has not been used. Please see https://github.com/pykeen/pykeen/blob/68af86ea387cd3282a7561884aee96d97e63dbda/src/pykeen/hpo/hpo.py#L801-L821

Reply:
is this relevant for this PR though? could you split it out into a separate one if not?

Reply:
oh i see below. I'm still a bit confused though